cmake/thirdparty/SetupSeracThirdParty.cmake

```diff
@@ -159,6 +159,7 @@ if (NOT SERAC_THIRD_PARTY_LIBRARIES_FOUND)
 
         find_package(caliper REQUIRED NO_DEFAULT_PATH 
                      PATHS ${CALIPER_DIR})
+        list(APPEND CALIPER_INCLUDE_DIRS ${CALIPER_DIR}/include)
```

Comment:
I'm confused.. does MFEM use this variable in the codevelop build? Shouldn't this be added to the `caliper` target?

Reply:
Yes. mfem uses that variable. this is the error when i remove the path. a reproducer is in #1095

```
/usr/WS2/meemee/serac/repo/mfem/fem/integ/../../general/annotation.hpp:19:10: fatal error: 'caliper/cali.h' file not found
#include <caliper/cali.h>
         ^~~~~~~~~~~~~~~~
```

I pushed this really quickly the other day. i have a slightly nicer solution ill be uploading right now

Reply:
Okay, glad you found this. A comment explaining would be helpful.

Reply:
This is where the variable is used in mfem. This might be happening since Caliper is an interface target?

https://github.com/mfem/mfem/blob/master/CMakeLists.txt#L560

Reply:
Ill update my comment to be a bit more detailed.